packages/babel-preset-expo/__tests__/index-test.js

```diff
@@ -72,3 +72,48 @@ import 'react-native-vector-icons';
   expect(code).toMatch(/"@expo\/vector-icons"/);
   expect(code).toMatchSnapshot();
 });
+
+it(`uses lazy - default (no option) matches option "null"`, () => {
+  let testFilename = path.join(path.resolve(__dirname, 'samples'), 'Lazy.js');
+  let optionsDefault = {
+    babelrc: false,
+    presets: [preset],
+  };
+  let { codeDefault } = babel.transformFileSync(testFilename, optionsDefault);
+
+  let optionsNull = {
+    babelrc: false,
+    presets: [[preset, { lazy: null }]],
+  };
+  let { codeNull } = babel.transformFileSync(testFilename, optionsNull);
+
+  expect(codeDefault).toEqual(codeNull);
+});
+
+it(`uses lazy - different values`, () => {
```

Comment:
For a test like this where the code is the same and the inputs are just slightly different, you can use:
```
describe.each(
   [[null],
    [false],
    [true],
    [['inline-comp', './inline-func', '../inline-func-with-side-effects.fx.ts']],
    [name => !(name.endsWith('.fx') || name.endsWith('.fx.js') || name.endsWith('.fx.ts'))],
  ])(`accepts %p`, lazyOption => {
  ... test
});
```
```